auth/postgres/key.go

```diff
@@ -17,9 +17,13 @@ var (
 )
 var _ auth.KeyRepository = (*repo)(nil)
 
+// Postgres error codes:
+// https://www.postgresql.org/docs/current/errcodes-appendix.html
 const (
-	errDuplicate = "unique_violation"
-	errInvalid   = "invalid_text_representation"
+	errDuplicate = "23505" // unique violation
+	errInvalid   = "22P02" // invalid input value
+	errFK        = "23503" // foreign key violation
+	errTooLong   = "22001" // varchar size violation
```

Comment:
Does it used ?

Reply:
Yes, but in `groups.go`. To avoid confusion, I moved PQ error codes to `init.go` for all services.